public/kibi_timeline.js

```diff
@@ -203,68 +203,87 @@ define(function (require) {
               startRawFieldValue = hit.fields[params.startField];
 
               let endFieldValue = null;
+              if (params.endField) {
+                if (params.endFieldSequence) { // in kibi, we have the path property of a field
+                  endFieldValue = kibiUtils.getValuesAtPath(hit._source, params.endFieldSequence);
+                } else {
+                  endFieldValue = _.get(hit._source, params.endField);
+                }
+                endRawFieldValue = hit.fields[params.endField];
 
-              if (startFieldValue && (!_.isArray(startFieldValue) || startFieldValue.length)) {
-                if (timelineHelper.isMultivalued(startFieldValue)) {
-                  detectedMultivaluedStart = true;
+                if (endFieldValue.length !== startFieldValue.length) {
+                  notify.warning('Check your data - the number of values in the \'Event end date\'' +
```

Comment:
Use the flag `$scope.notifyDataErrors` for printing or not this message.

Also, improve the message by passing the following information:
- name of the start/end fields
- ID of the document where the problem is